inlong-agent/agent-plugins/src/main/java/org/apache/inlong/agent/plugin/fetcher/ManagerFetcher.java

```diff
@@ -154,7 +154,7 @@ private String buildBaseUrl() {
     /**
      * build vip url for manager according to config
      *
-     * example - http://127.0.0.1:8080/api/inlong/manager/openapi/agent/getInLongManagerIp
+     * example - http://127.0.0.1:8080/manager/openapi/agent/getInLongManagerIp
```

Comment:
The `getInLongManagerIp` already changed, please change to `getManagerIpList`, thanks.